internal/ethapi/backend.go

```diff
@@ -124,6 +124,9 @@ func GetAPIs(apiBackend Backend, chain *core.BlockChain) []rpc.API {
 		}, {
 			Namespace: "eth",
 			Service:   NewBundleAPI(apiBackend, chain),
+		}, {
+			Namespace: "enclave",
```

Comment:
the idea is to have a namespace that can be exposed through the SGX SSL setup. wonder whether it'd fit better into the patch rather than in the codebase 🤔 

```suggestion
			Namespace: "protect",
```

Reply:
I can add this another patch to the geth-sgx-gramine repo, however that would be the first builder-only patch introduced there. No strong opinion, happy to follow your decision.